test/integration_test.go

```diff
@@ -30,6 +31,8 @@ func TestIntegration_Value_Simple(t *testing.T) {
 	dir, err := os.MkdirTemp(os.TempDir(), "dela-integration-test")
 	require.NoError(t, err)
 
+	to := time.Second * 10 // transaction inclusion timeout
```

Comment:
given this variable is used 70+ lines down, I think it's not a waste to call it `timeout` in full 😁